source/includes/Patient/patients.md

```diff
@@ -303,6 +303,94 @@ fetch("/openmrs/ws/rest/v1/patient", requestOptions)
+      "gender":"M",
+      "age":47,
+      "birthdate":"1970-01-01T00:00:00.000+0100",
+      "birthdateEstimated":false,
+      "dead":false,
+      "deathDate":null,
+      "causeOfDeath":null,
+      "names":[
+         {
+            "givenName":"Thomas",
+            "familyName":"Smith"
+         }
+      ],
+      "addresses": [
+        {
+        "address1": "30, Vivekananda Layout, Munnekolal,Marathahalli",
+        "cityVillage": "Bengaluru",
+        "country": "India",
+        "postalCode": "560037"
+        }
+    ]
+   }
+}
+
+```
+
+
+```java
+
+OkHttpClient client = new OkHttpClient().newBuilder()
+  .build();
+MediaType mediaType = MediaType.parse("application/json");
+RequestBody body = RequestBody.create(mediaType, "{\r\n \"identifiers\":[ \r\n  {\"identifier\":\"103VWY7\", \r\n \"identifierType\":\"71075074-f02e-4270-89a3-f2dcda436f70\",\r\n \"location\":\"9356400c-a5a2-4532-8f2b-2361b3446eb8\", \r\n \"preferred\":true}\r\n ], \r\n \"person\":{\r\n \"gender\":\"M\", \r\n \"age\":47,\r\n \"birthdate\":\"1970-01-01T00:00:00.000+0100\",\r\n \"birthdateEstimated\":false,\r\n \"dead\":false,\r\n \"deathDate\":null,\r\n \"causeOfDeath\":null,\r\n \"names\":[\r\n {\r\n  \"givenName\":\"Thomas\",\r\n \"familyName\":\"Smith\" \r\n }\r\n ],\r\n \"addresses\":[\r\n {\r\n \"address1\":\"30,VivekanandaLayout,Munnekolal,Marathahalli\",\r\n \"cityVillage\":\"Bengaluru\",\r\n \"country\":\"India\",\r\n \"postalCode\":\"560037\"\r\n }\r\n]\r\n}\r\n }");
+Request request = new Request.Builder()
+  .url("/openmrs/ws/rest/v1/patient")
+  .method("POST", body)
+  .addHeader("Authorization", "Basic YWRtaW46QWRtaW4xMjM=")
+  .addHeader("Content-Type", "application/json")
+  .addHeader("Cookie", "JSESSIONID=24D0761924138ED7E55C2CB6806B0633")
+  .build();
+Response response = client.newCall(request).execute();
+
+```
+
+```javascript
+
+var requestHeaders = new Headers();
+requestHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+requestHeaders.append("Content-Type", "application/json");
+requestHeaders.append("Cookie", "JSESSIONID=24D0761924138ED7E55C2CB6806B0633");
+
+var raw = JSON.stringify({"identifiers":[{"identifier":"103VWY7","identifierType":"71075074-f02e-4270-89a3-f2dcda436f70","location":"9356400c-a5a2-4532-8f2b-2361b3446eb8","preferred":true}],"person":{"gender":"M","age":47,"birthdate":"1970-01-01T00:00:00.000+0100","birthdateEstimated":false,"dead":false,"deathDate":null,"causeOfDeath":null,"names":[{"givenName":"Thomas","familyName":"Smith"}] , "addresses": [{"address1": "30, Vivekananda Layout, Munnekolal,Marathahalli","cityVillage": "Bengaluru","country": "India","postalCode": "560037"}]}});
+
+var requestOptions = {
+  method: 'POST',
+  headers: requestHeaders,
+  body: raw,
+  redirect: 'follow'
+};
+
+fetch("/openmrs/ws/rest/v1/patient", requestOptions)
+  .then(response => response.text())
+  .then(result => console.log(result))
+  .catch(error => console.log('error', error));
+
+```
+
+
 ## Create a patient
```

Comment:
I know this isn't directly related to your PR, but, since you're here, could you change this line to
`## Update a patient`
which is what I believe the next section documents?

Reply:
Sure. I will do it. I have another suggestion. While trying to follow the instructions on the Readme of this repository, I encountered a problem with  starting the documentation server using docker. I had capital letters in my file path:

wrong : `~/Webdev/Openmrs/openmrs-contrib-rest-api-docs` 
correct : `~/webdev/openmrs/openmrs-contrib-rest-api-docs`

Which gave me an error, but the error message was not clear. 

It would be better if there is a small note warning against using file paths with capital letters.

Reply:
Thanks! 👍 I think your problem wasn't related to the case of your path; rather, the lack of quotes in the command to run the docker image. Based on your feedback, I've [updated the README](https://github.com/openmrs/openmrs-contrib-rest-api-docs/commit/3fb6256cf986518459d154b2d3c113240503a327) (don't forget to pull before updating your PR 🙂).